DEV_TO_DO.md

```diff
@@ -0,0 +1,79 @@
+# Development To-Do List
+
+## Monorepo
+
+- Get `nx format:check` and `nx format:write` to work like `prettier -c .` and `prettier -w .`
```

Comment:
FYI if you start each item with `- [ ] ` instead of just `- ` they become checkboxes.

Reply:
Good thought. I'd prefer to just delete lines in this file when they're not longer needed, and the checkboxes might encourage using the file for longer than is prudent rather than migrating to a issue/kanban-based workflow.